blessclient/client.py

```diff
@@ -17,19 +17,24 @@
 import copy
 import subprocess
 import json
-from random import randint
 
-import awsmfautils
-import tokengui
-from bless_aws import BlessAWS
-from bless_cache import BlessCache
-from user_ip import UserIP
-from bless_lambda import BlessLambda
-from bless_config import BlessConfig
-from lambda_invocation_exception import LambdaInvocationException
+import six
+
+from . import awsmfautils
+from .bless_aws import BlessAWS
+from .bless_cache import BlessCache
+from .user_ip import UserIP
+from .bless_lambda import BlessLambda
+from .bless_config import BlessConfig
+from .lambda_invocation_exception import LambdaInvocationException
 
 import logging
 
+try:
+    from . import tokengui
+except ImportError:
+    tokengui = None
```

Comment:
The reason for the token gui is because we can't get stdin during an SSH call. Can you move the ImportError to the tokengui file, and if the import fails, set a flag? And then when `get_mfa_token_gui` is called, if that flag is set, then raise an error (e.g. "Token GUI failed to load. Run bless without the `--gui` flag")

Reply:
I check this value later on in this file -- so the implementation is equivalent to moving the `ImportError` -- I can however add an exception in the case that the token gui is unavailable and stdin is not a tty which I think will satisfy your concerns :)